profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_dgt_connector/plugins/tmgmt_dgt_connector.plugin.inc

```diff
@@ -0,0 +1,284 @@
+<?php
+
+/**
+ * @file
+ * Provides TMGMT DGT Connector translation plugin controller.
+ */
+
+use EC\Poetry\Poetry;
+use EC\Poetry\Messages\Components;
+use EC\Poetry\Exceptions;
+
+/**
+ * DGT Connector translation plugin controller.
+ *
+ * It's now mostly based on TMGMTPoetryTranslatorPluginController
+ * Later this should extend TMGMTDefaultTranslatorPluginController and
+ * TMGMTPoetryTranslatorPluginController should be deprecated.
+ */
+class TmgmtDgtConnectorTranslatorPluginController extends TMGMTPoetryTranslatorPluginController {
```

Comment:
Why not trying to make extending `TMGMTPoetryTranslatorPluginController` from `TmgmtDgtConnectorTranslatorPluginController` instead, so, when the old plugin will be removed , we have just to delete it and nothing is to touch in the new plugin?

Reply:
After third thoughts, and as discussed with @gillesdeudon and @Fefaine , I will maintain TmgmtDgtConnectorTranslatorPluginController extending TMGMTPoetryTranslatorPluginController, to minimize the changes in old code and duplication of code.

I have added a section in the [epic](https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-869) to refactor these classes in a final stage.